tchannel/collector/app/flags.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"flag"
+
+	"github.com/spf13/viper"
+)
+
+const (
+	// CollectorTChannel is the default port for TChannel server for sending spans
+	CollectorTChannel = 14267
+	collectorPort     = "collector.port"
```

Comment:
It looks like we're going back to the deprecated flag name? Maybe we can remove the old one, as it's been quite a while since we deprecated it.

Reply:
The intent of this PR is to just move tchannel code to a separate package. The component itself will be removed in the follow-up PR along with flags bindings.

Reply:
nvm, I though we had another option similar to
```
      --collector.grpc-port int                         The gRPC port for the collector service (default 14250)
```